src/Webapi/Webapi__FormData.res

```diff
@@ -12,6 +12,8 @@ module EntryValue = {
 type t
 
 @new external make: unit => t = "FormData"
+@new external fromForm: Webapi__Dom__HtmlFormElement.t => t = "FormData"
```

Comment:
```suggestion
@new external makeWithFormData: Webapi__Dom__HtmlFormElement.t => t = "FormData"
```

I think we should stick to how other `make` methods are styled:
https://github.com/tinymce/rescript-webapi/blob/44a621dcb886107988ed08a048597ff7a743540c/src/Webapi/Webapi__Fetch.res#L274

Reply:
Done, but instead I used `makeWithHtmlForm` because the function takes a `HtmlFormElement.t` and returns a `FormData.t`.

Reply:
Right, yes, of course. Thank you!